bin/zopen-importenvs

```diff
@@ -20,6 +20,22 @@ fi
 
 #depsPath has the details of the location of ports whose env files are consumed for sourcing 
 depsPath="$HOME/zopen/prod|$HOME/zopen/boot"
+configFile=~/.zopen-config
+if [ -f "$configFile" ]
+then
```

Comment:
It might be simpler to source . "$HOME/.zopen-config", as in:
https://github.com/ZOSOpenTools/utils/blob/main/bin/lib/common.inc#L159